patches/server/0969-Added-the-ability-to-disable-non-player-map-markers.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: poonkje112 <aaron.knoop@live.nl>
+Date: Sun, 26 Mar 2023 04:17:53 +0200
+Subject: [PATCH] Added the ability to disable non-player map markers
+
+
+diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+index 8d442c5a498ecf288a0cc0c54889c6e2fda849ce..772cc473de64076dcb11b84f48fa08ab1d7ef167 100644
+--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+@@ -286,5 +286,6 @@ public class GlobalConfiguration extends ConfigurationPart {
+         public boolean lagCompensateBlockBreaking = true;
+         public boolean useDimensionTypeForCustomSpawners = false;
+         public boolean strictAdvancementDimensionCheck = false;
++        public boolean showMapMarkers = true;
```

Comment:
This bit of diff should go in the patch which adds this file. We try to keep all configuration options in the same patch, where the use of those options can be its own patch. It should be one of the very first server patches.

Reply:
I totally missed that part in the CONTRIBUTING.MD
It should be resolved now.